File: SisteCredito/SisteCredito-Core/Models/Usuarios.cs
using Microsoft.AspNetCore.Identity;

namespace SisteCredito_Core.Models;

public class Usuarios : IdentityUser
{
    public string Nombres { get; set; } = string.Empty;
    public string Apellidos { get; set; } = string.Empty;
    public string Rol { get; set; } = string.Empty;

}

File: SisteCredito-PruebaTecnica-FrontEnd/src/app/Models/usuarios.ts
export interface usuarios{
  id: string | null,
  nombres: string,
  apellidos: string,
  rol: string,
  email: string,
  userName: string,
  token: string,
  password: string
}

export interface usuariosLoginDto{
  email: string,
  password: string
}

export interface usuarioRegisterDto {
  nombres: string,
  apellidos: string,
  rol: string | null,
  email: string | null,
  userName: string | null,
  password: string | null
}
